src/changes/changes.xml

```diff
@@ -23,6 +23,12 @@
     <author email="roland@jolokia.org">Roland Huß</author>
   </properties>
   <body>
+    <release version="1.6.1-SNAPSHOT" description="Release 1.6.1-SNAPSHOT" date="yyyy-mm-dd">
+      <action dev="nevenr" type="add">
```

Comment:
just add a `issue=""` attribute here, so you don't have to reference it in the comment (see other entries for examples)